decentraland/unity-renderer
# style: fix profile card scroll bar texture and collectibles scroll view mask

This PR fixes
- Texture used for the handle of the scroll bar and erases the tint animation.
- Collectibles section scroll view max heigh size increase to reduce the gap between the section buttons and the content bellow.

BEFORE
<img width="1310" alt="Screenshot 2023-06-30 at 15 12 01" src="https://github.com/decentraland/unity-renderer/assets/51088292/2c8e01d0-7ce6-41c5-a30f-b5a2e649f4f3">


AFTER
<img width="1278" alt="Screenshot 2023-06-30 at 15 15 28" src="https://github.com/decentraland/unity-renderer/assets/51088292/bddeeb32-bf01-4a4c-8ec0-bbb83b10c568">

### **How to test**
1- Go to https://play.decentraland.org/?explorer-branch=style/ProfileCardPolish
2- Open the profile card of any user, ideally with many collectibles and with a wide description. 
3- Check the scroll bar doesn't look clunky or has any tint animation with mouse interaction, in both sections.
4- Go to the `collectibles` section and check the scroll bar is anchored in X axis in the same position than in `About`.
